terraform/10-video-hearing-wowza/20-main.tf

```diff
@@ -13,6 +13,11 @@ data "azurerm_resource_group" "vh-infra-core" {
 # VH - Key Vault Lookup
 #--------------------------------------------------------------
 
+data "azurerm_key_vault_secret" "dynatrace_token" {
```

Comment:
Can we do a lookup in the keyvault vars instead of doing a data call?

Reply:
don't get what you mean, the dynatrace token is handled via the 09-video-hearing-core directory.

Reply:
ahh sorry. didn't realise the file name